Src/Newtonsoft.Json-for-Unity.Tests/Assets/Aot.Tests/AotTests.cs

```diff
@@ -5,46 +5,52 @@
 
 namespace Aot.Tests
 {
+    /// <summary>
+    /// Tests are only relevant for AOT builds, such as when using IL2CPP
+    /// </summary>
     [TestFixture]
     public class AotTests
     {
-#if ENABLE_IL2CPP
         static AotTests()
         {
-            var myAotEnsuredList = new List<MyAotEnsuredClass>();
+            // Ensure it's generated
+            _ = new List<MyAotEnsuredClass>();
         }
 
-        class MyNonAotClass
+        private class MyNonAotClass
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private type 'MyNonAotClass'.](https://app.codacy.com/app/jilleJr/Newtonsoft.Json-for-Unity/pullRequest?prid=4554595)